src/components/TeamMemberList.js

```diff
@@ -1,31 +1,41 @@
-import { useState, useEffect } from "react";
+import { useState, useEffect, useCallback } from "react";
 import TeamMember from "./TeamMember";
 import axios from "axios";
+import apiUrl from "../api/paths";
 
 const TeamMemberList = (props) => {
   const [users, setUsers] = useState([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
 
-  useEffect(() => {
-    const fetchUsers = async () => {
-      const response = await axios.get(process.env.REACT_APP_ENDPOINT_URL);
+  const fetchUsers = useCallback(async () => {
```

Comment:
just a tip: if you don't want useCallback (because for example code looks less readable) you can move this method inside useEffect, it will be rendered only once because empty array of dependencies

Reply:
Removed useCallback for better visibility and removed dependencies from useEffect.